.eslintrc.json

```diff
@@ -1,71 +1,66 @@
 {
-    "globals": {
-      "JSX": true,
-      "Office": "readonly",
-      "Word": "readonly",
-      "OfficeExtension": "readonly"
+  "globals": {
+    "JSX": true,
+    "Office": "readonly",
+    "Word": "readonly",
+    "OfficeExtension": "readonly"
+  },
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es2021": true
+  },
+  "extends": [
+    "plugin:react/recommended",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:office-addins/react",
+    "airbnb",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:prettier/recommended"
```

Comment:
I also used the prettier plugin first, but in the end it was a hassle. Also you don't want to get warnings in your editor that disappear once you save. In the end, I went the recommended path of splitting eslint and prettier, which works really nicely. See https://github.com/JabRef/JabRefOnline/pull/1315 for reference.

Reply:
Will do it in a follow-up. Thanks for your quick response!